starlite/types/protocols.py

```diff
@@ -1,57 +1,178 @@
-from typing import Any
+from typing import Any, overload
 
 from typing_extensions import Protocol
 
 
 class Logger(Protocol):  # pragma: no cover
+    @overload
     def debug(self, event: str, **kwargs: Any) -> Any:
         """Outputs a log message at 'DEBUG' level.
 
         Args:
              event: Log message.
              **kwargs: Any kwargs.
         """
+        ...
 
+    @overload
+    def debug(self, event: str, *args: Any) -> Any:
+        """Outputs a log message at 'DEBUG' level.
+
+        Args:
+             event: Log message.
+             **kwargs: Any kwargs.
```

Comment:
Shouldn't it be `*args`?

Reply:
Good catch.  I've pushed updates to all of the docstrings.